CMakeLists.txt

```diff
@@ -325,7 +327,7 @@ if (StbImage_FOUND)
 
   # Make sure that we can link STB in multiple translation units.
   include(CMake/TestStaticSTB.cmake)
-  if (NOT CMAKE_HAS_WORKING_STATIC_STB)
+  if (NOT CMAKE_HAS_WORKING_STATIC_STB AND NOT CMAKE_CROSSCOMPILING)
```

Comment:
Why disable this message when cross-compiling?  We do need to ensure the version of STB is sufficiently new that it can link across multiple translation units, even in the cross-compilation context (since you can still have multiple translation units in a single program).

Reply:
because when using musl, even the newly downloaded version using the autodownloader is throwing an error saying that it can not link across multiple translation units. This does not make sense, and if you have already downloaded it manually it is throwing this error when using g++. Therefore, I have stopped checking for now in order to understand what is happening. However, I can put it back, it will always compile on the CI

Reply:
It might be throwing a different error.  Take a look at `CMakeOutput.log` to see what the actual error is.  The file that runs the test is `CMake/TestStaticSTB.cmake`.  Really that error message will be thrown whenever a simple STB program fails to compile, so, the output could mask a deeper issue.

Reply:
this error just disappeared, so probably it was a network error or something, but now I did not see it again at all.